libs/langchain/langchain/retrievers/kendra.py

```diff
@@ -336,13 +345,25 @@ class AmazonKendraRetriever(BaseRetriever):
     page_content_formatter: Callable[[ResultItem], str] = combined_text
     client: Any
     user_context: Optional[Dict] = None
+    score_confidence: float = 0
+    score_confidence_dict = {
+        "NOT_AVAILABLE": 0,
+        "LOW": 0.25,
+        "MEDIUM": 0.50,
+        "HIGH": 0.75,
+        "VERY_HIGH": 1,
```

Comment:
Nit: can we make ints to floats i.e. 0.0 and 1.0 instead of 0 and 1?

Reply:
Also I think this dictionary should be outside the class otherwise it just gets recreated for each object.

Reply:
Something like above the AmazonKendraRetriever class:

```python
KENDRA_CONFIDENCE_MAPPING = {
        "NOT_AVAILABLE": 0.0,
        "LOW": 0.25,
        "MEDIUM": 0.50,
        "HIGH": 0.75,
        "VERY_HIGH": 1.0,
    }
```